pkg/controller/render/render_controller.go

```diff
@@ -416,6 +460,76 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 		return nil
 	}
 
+	// If it's a pool named "layered", it's special, designate it automatically
+	// TODO(jkyros): take this out when we decide actual UX
+	if pool.Name == "layered" {
```

Comment:
It feels more flexible to have the opt-in mechanism be adding the label, WDYT?  Then there's no special pool names, only a special label.

Reply:
I left that in as a "just make it go" demo optimization for people playing with it so people could just make it work with one step. 

I did rework it under the hood so that everywhere but right there it's a label on the pool (right below that block, all it does is "add the layered label to the pool"). If you slap ""machineconfiguration.openshift.io/layered" on your worker pool it will regard it as "layered" and it will work  (ok your nodes don't boot, so don't do that right now unless you want to boot them manually, but you know what I mean).

Reply:
When I read over this I also wondered if it would make more sense to add `|| pool.Name == "layered"` as a condition in `IsLayeredPool` but probably not worth changing if this is temporary